### PR TITLE
Cherry-pick fixing Xcode 13 beta 3 build issues (#8405)

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

**Comment by paulb777** [comment]
Propose disregarding the check failures to avoid back-porting clang-format 12.0.1 changes to the release branch

File: FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessagingRendering.h [hunk 1]
```diff
@@ -174,6 +178,7 @@ NS_SWIFT_NAME(InAppMessagingDisplayMessage)
 @end
 
 NS_SWIFT_NAME(InAppMessagingCardDisplay)
+NS_EXTENSION_UNAVAILABLE_IOS("Firebase In App Messaging is not supported for iOS extensions.")
```
Comment (ryanwilson) [inline FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessagingRendering.h:181]: Nit: for consistency can we put this above NS_SWIFT_NAME?

**Review by ryanwilson — APPROVED** [review]
One nit then LGTM

File: FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessagingRendering.h (hunk 1 above)
Comment (ryanwilson) [reply FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessagingRendering.h:180]: Thanks!

**Pull request merged**